tests/dialects/test_snowflake.py

```diff
@@ -983,6 +983,16 @@ def test_snowflake(self):
         # Test BITOR with three arguments, padding on the left
         self.validate_identity("SELECT BITOR(a, b, 'LEFT') FROM table_name")
 
+        # Test BITXOR with three arguments
+        self.validate_identity("SELECT BITXOR(a, b, 'LEFT') FROM table_name")
```

Comment:
Generally we can make the tests less noisy by:
- Removing the comments, they're not really required here
- Trimming down the SQL e.g we can even test the functions as standalone expressions or as simple `SELECT`s